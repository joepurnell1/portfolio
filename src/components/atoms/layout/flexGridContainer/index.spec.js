import React from 'react';
import renderer from 'react-test-renderer';
import FlexGridContainer from './index';

describe('flex container component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <FlexGridContainer>Will you stand above me</FlexGridContainer>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render with given maxWidth as expected', () => {
    const render = renderer.create(
      <FlexGridContainer maxWidth="500px">Look my way, never love me</FlexGridContainer>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
