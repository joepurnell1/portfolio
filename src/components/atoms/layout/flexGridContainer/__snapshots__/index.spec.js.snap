// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`flex container component should render default as expected 1`] = `
<div
  className="sc-bdVaJa hEFZfD"
>
  Will you stand above me
</div>
`;

exports[`flex container component should render with given maxWidth as expected 1`] = `
<div
  className="sc-bdVaJa fkMsVG"
>
  Look my way, never love me
</div>
`;
